Extensions/BackButton.json

```diff
@@ -0,0 +1,101 @@
+{
+  "author": "Arthur Pacaud (arthuro555)",
+  "description": "This extension prevents the back button from quitting the game and adds a condition to decide what it does instead.",
```

Comment:
"Prevents the back button from quitting the game and add a condition to check when it's pressed (to allow customising its behavior)."

(I avoid saying "This extension", as it's implied when shown on screen)